test/functional/terminal/tui_spec.lua

```diff
@@ -2003,6 +2003,39 @@ describe('TUI', function()
     ]])
   end)
 
+  it('invalidated regions are cleared with terminal background attr', function()
+    local screen = Screen.new(50, 10)
+    screen:set_default_attr_ids({ [1] = { foreground = Screen.colors.Black } })
```

Comment:
Added a test that at least fails on master, although I'm not sure if it tests for the behavior correctly. Presumably `Screen.colors.Black` **is** `normal_bg`, in which case it does. Could also mean that the test still produces an unwanted black screen here? I can no longer reproduce that locally.